internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -797,9 +797,18 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	var publishers template.PublishOpts
 	// convert the topics to template Topics
 	for _, topic := range topics {
+		var fifo template.Fifo
```

Comment:
can we rename all references to `Fifo` to be either `fifo` or `FIFO`? 🥺 

https://github.com/golang/go/wiki/CodeReviewComments#initialisms